src/components/Modal/jsonModal.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+import './style.scss';
+
+export default class JSONModal extends React.Component {
+  constructor(props) {
+    super(props);
+    const { attendees } = this.props;
+    this.state = {
+      attendees: JSON.parse(attendees).members,
+    };
+  }
+
+  render() {
+    const { title, style, onChange } = this.props;
+    const { attendees } = this.state;
+    return (
+      <div className="modal-wrapper">
+        <div className="modal-container" style={style || null}>
+          <div style={{ padding: '30px' }}>
+            <h1>{title}</h1>
+            <br />
+            <div className="modal-table">
+              <table style={{ width: '100%' }}>
+                <thead>
+                  <tr>
+                    <td />
+                    <td>Name</td>
+                    <td>Year</td>
+                    <td>Major</td>
+                  </tr>
+                </thead>
+                <tbody>
+                  {
+                    attendees.map(member => (
+                      <tr>
+                        <td><img src={member.picture} alt="Attendee" /></td>
+                        <td>{`${member.firstName} ${member.lastName}`}</td>
```

Comment:
Redundant string construction
```suggestion
                        <td>${member.firstName} ${member.lastName}</td>
```